fhevm/params.go

```diff
@@ -73,11 +73,13 @@ func DefaultGasCosts() GasCosts {
 			tfhe.FheUint64: 178000 + AdjustFHEGas,
 		},
 		FheDecrypt: map[tfhe.FheUintType]uint64{
-			tfhe.FheUint4:  500000,
-			tfhe.FheUint8:  500000,
-			tfhe.FheUint16: 500000,
-			tfhe.FheUint32: 500000,
-			tfhe.FheUint64: 500000,
+			tfhe.FheBool:     500000,
+			tfhe.FheUint4:    500000,
+			tfhe.FheUint8:    500000,
+			tfhe.FheUint16:   500000,
+			tfhe.FheUint32:   500000,
+			tfhe.FheUint64:   500000,
```

Comment:
You forgot to add FHEUint160 here

Reply:
I removed decrypt and reencrypt altogether.